### DISCUSSION TITLE
How to proceed , if i want change the backbone to efficentNet encoder?

### DISCUSSION THREAD

**Discussion by Ab-hinav** [bowenc0221/panoptic-deeplab#105] [Q&A]
I am trying to change the backbone

**Comment by bowenc0221** [comment]
You can write a wrapper function like this: https://github.com/bowenc0221/panoptic-deeplab/blob/cf8e20bbbf1cf540c7593434b965a93c4a889890/tools_d2/d2/backbone.py#L12-L13 to register your backbone in Detectron2.

And you can follow config like this: https://github.com/bowenc0221/panoptic-deeplab/blob/cf8e20bbbf1cf540c7593434b965a93c4a889890/tools_d2/configs/Cityscapes-PanopticSegmentation/panoptic_deeplab_H_48_os16_mg124_poly_90k_bs32_crop_512_1024_dsconv.yaml#L6-L7 to change your backbone

**Comment by lucasjinreal** [comment]
@bowenc0221  how's the panoptic -deeplab performance compare with panoptic-fpn?
